Makefile

```diff
@@ -324,3 +325,8 @@ golint: get-ci-tools
 	PATH=$(GOBIN):$(PATH); $(CI_TOOLS_REPO_DIR)/test-runner/golint.sh
 	PATH=$(GOBIN):$(PATH); $(CI_TOOLS_REPO_DIR)/test-runner/golint.sh ./api
 
+.PHONY: gowork
```

Comment:
this is already defiend on line 131

https://github.com/openstack-k8s-operators/cinder-operator/pull/123/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R131

Reply:
done